src/pages/Dashboard/Markets/SupplyMarket/index.tsx

```diff
@@ -135,22 +142,24 @@ const SupplyMarket: React.FC = () => {
   const toggleAssetCollateral = (asset: Asset) => {
     if (!account) {
       throw new ToastError(
-        'Account Required',
-        'Please connect your wallet to set an asset as collateral.',
+        t('markets.errors.accountError.title'),
+        t('markets.errors.accountError.description'),
       );
     } else if (!asset || !asset.borrowBalance.isZero()) {
       throw new ToastError(
-        'Collateral Required',
-        'Please repay all borrowed assets or set other assets as collateral.',
+        t('markets.errors.collateralRequired.title'),
+        t('markets.errors.collateralRequired.description'),
       );
     } else if (!asset.collateral) {
       try {
         setConfirmCollateral(asset);
         enterMarkets({ vtokenAddresses: [asset.vtokenAddress], account });
       } catch (error) {
         throw new ToastError(
-          'Collateral Enable Error',
-          `There was a problem enabeling collateral ${asset.name}. Please try again.`,
+          t('markets.errors.collateralEnableError.title'),
+          `${t('markets.errors.collateralEnableError.description')} ${asset.name}. ${t(
+            'markets.errors.tryAgain',
+          )}`,
```

Comment:
Same thing here, the line needs to be one full sentence. You can use [interpolation](https://react.i18next.com/legacy-v9/interpolate) to insert `asset.name` within the sentence:
```typescript
t('markets.errors.collateralEnablingError', { assetName: asset.name })
```

Then in your JSON file the translation will be like that:
```json
"collateralEnablingError": "There was a problem enabling {{assetName}} as a collateral. Please try again."
```